src/components/layermanager/LayermanagerDirective.js

```diff
@@ -57,7 +57,7 @@ goog.require('ga_window_service');
     // Timestamps list template
     var tpl =
       '<div class="ga-layer-timestamps">' +
-        '<div tabindex="1" ng-if="tmpLayer.type == \'wms\'" ' +
+        '<div tabindex="1" ng-if="tmpLayer.timeBehaviour == \'all\'" ' +
```

Comment:
not sure if this is really equivalent.

Reply:
I've verified in the LayersConfig . All the layer with `timeBehaviour='all'`  are wms